anyblok_furetui/furetui/furetui.py

```diff
@@ -198,3 +199,42 @@ def apply_value(value):
             res = getattr(obj, call)(**options, **data)
 
         return res
+
+    @classmethod
+    def validate_resources(cls):
+        res = []
+        res.extend(cls.validate_form_resources())
+        res.extend(cls.validate_list_resources())
+        return res
+
+    @classmethod
+    def validate_form_resources(cls):
+        res = []
+        Form = cls.registry.FuretUI.Resource.Form
+        for resource in Form.query().filter(Form.template.isnot(None)):
+            try:
+                resource.get_definitions()
+            except ResourceTemplateRendererException as e:
+                logger.error(str(e))
+                res.append(e)
+            except Exception as e:
+                logger.exception(str(e))
+                raise
```

Comment:
I supposed it would be nice to `raise e` instead nothing

Reply:
only raise reraise the same exception

```
>>> try:
...     raise KeyError("An exception")
... except Exception as e:
...     print(str(e))
...     raise
... 
'An exception'
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
KeyError: 'An exception'

```